src/widgets.tsx

```diff
@@ -327,4 +388,37 @@ export const ProfileImage = (props:{style?:ImageStyle}):JSX.Element => {
         </>
 
     );
+}
+
+export const BackButton = (props:{callback:(() => void)}):JSX.Element => {
+    const styles = StyleSheet.create({
+        backButton: {
+            justifyContent: "center",
+            alignItems: "center",
+            height: 55,
+            width: 55,
+            borderRadius: 15,
+          },
+          backButtonView: {
+            position: "absolute",
+            top: 1,
+            left: 1
+          },
+    })
+
+    return (
+        <View style={styles.backButtonView}>
+            <TouchableOpacity
+                onPress={() => props.callback()}
```

Comment:
Can we have a default behavior like `pop()` from the page stack?  Then `props.callback()` could be optional, but would override if provided.  If not possible and context is needed; then never mind.